__tests__/app.test.js

```diff
@@ -211,6 +214,111 @@ describe('app', () => {
+                    review_img_url: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: 105
+                    }
+                expect(updatedReview).toMatchObject(expectedReview)
+                });
+        });
+        test('Responds 200 OK and sends back the updated review for negative numbers in the inc_votes', () => {
+            return request(app).patch('/api/reviews/1')
+            .send({ inc_votes: -100})
+            .expect(200)
+            .then((response) => {
+                const updatedReview = response.body.updatedReview
+                const expectedReview = {
+                    review_id: 1,
+                    title: expect.any(String),
+                    category: expect.any(String),
+                    designer: expect.any(String),
+                    owner: expect.any(String),
+                    review_body: expect.any(String),
+                    review_img_url: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: -99
+                }
+                expect(updatedReview).toMatchObject(expectedReview);
+            });
+        });
+        test('Update votes works when used multiple times', () => {
+            return request(app).patch('/api/reviews/1')
+            .send({ inc_votes: -100})
+            .expect(200)
+            .then(() => {
+                return request(app).patch('/api/reviews/1').send({inc_votes: 50}).expect(200)
+                .then((response) => {
+                    const updatedReview = response.body.updatedReview
+                    const expectedReview = {
+                        review_id: 1,
+                        title: expect.any(String),
+                        category: expect.any(String),
+                        designer: expect.any(String),
+                        owner: expect.any(String),
+                        review_body: expect.any(String),
+                        review_img_url: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: -49
+                    }
+                    expect(updatedReview).toMatchObject(expectedReview);
+                });
+            })
+        });
+        test('Responds 400 Bad Request! when passed an empty request body', () => {
+            return request(app).patch('/api/reviews/3')
+            .send({})
+            .expect(400)
+            .then((response) => {            
+                expect(response.text).toBe("Bad Request!")
+                });
+        });
+        test('Responds 400 Bad Request! when passed a request body with invalid patch fields', () => {
+            return request(app).patch('/api/reviews/3')
+            .send({ review_body: "Hello" })
+            .expect(400)
+            .then((response) => {            
+                expect(response.text).toBe("Bad Request!")
+                });
+        });
```

Comment:
Maybe alter your test on 281 and 289 to be 400s with custom error messages explaining that what has been sent is wrong? You would need to handle this in the controller/model and send a custom error message from there.